hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -59,3 +61,4 @@ objects:
     disabledControllers: "${{DISABLED_CONTROLLERS}}"
     globalPullSecretRef:
       name: ${GLOBAL_PULL_SECRET}
+    deleteProtection: ${ENABLE_DELETION_PROTECTION}
```

Comment:
This will fail validation, as "false" is not a valid value. The only valid non-empty value is "enabled".

Reply:
Ok, can I use empty string? If not, seems like there's no way to use the current pipeline design, then.

Reply:
Yes, empty string should be the same as omitted. If it is not, then the kubebuilder validation on the field should be adjusted to allow it.

Reply:
done